superset/migrations/versions/b92d69a6643c_rename_csv_to_file.py

```diff
@@ -15,23 +15,54 @@
 # specific language governing permissions and limitations
 # under the License.
 """rename_csv_to_file
-
 Revision ID: b92d69a6643c
 Revises: 32646df09c64
 Create Date: 2021-09-19 14:42:20.130368
-
 """
 
 # revision identifiers, used by Alembic.
 revision = "b92d69a6643c"
 down_revision = "32646df09c64"
 
-import sqlalchemy as sa
+import json
+import logging
+
 from alembic import op
+import sqlalchemy as sa
+from sqlalchemy import Column, Integer, Text
 from sqlalchemy.engine.reflection import Inspector
+from sqlalchemy.ext.declarative import declarative_base
+
+from superset import db
+
+Base = declarative_base()
+
+
+class Database(Base):
+
+    __tablename__ = "dbs"
+    id = Column(Integer, primary_key=True)
+    extra = Column(Text)
 
 
 def upgrade():
+    bind = op.get_bind()
+    session = db.Session(bind=bind)
+
+    for database in session.query(Database).all():
+        try:
+            extra = json.loads(database.extra)
+        except json.decoder.JSONDecodeError as ex:
```

Comment:
Just to confirm is this the only exception that `json.loads(…)` can raise?

Reply:
it could technically raise a TypeError if `database.extra` isn't one of str, bytes or bytearray